tools/compare_chains.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+
+import pickle
+import sys
+from pathlib import Path
+from typing import List
+
+from recursive_diff import recursive_diff
+
+from chia.types.full_block import FullBlock
+
+
+def load_chain(file_path: Path) -> List[FullBlock]:
+    try:
+        bytes_list = file_path.read_bytes()
+        block_bytes_list: List[bytes] = pickle.loads(bytes_list)
+        blocks: List[FullBlock] = []
+        for block_bytes in block_bytes_list:
+            blocks.append(FullBlock.from_bytes_unchecked(block_bytes))
+        print(f"loaded {file_path} with {len(blocks)} blocks")
+        return blocks
+    except EOFError as e:
+        raise ValueError(f'failed to load file "{file_path}"') from e
+
+
+try:
+    chain1 = load_chain(Path(sys.argv[1]))
+    chain2 = load_chain(Path(sys.argv[2]))
```

Comment:
I would just use click.  Feel free to close this comment instead of taking action.